### ISSUE TITLE
Batch download stops at each image due to DateTime error

### ISSUE THREAD

**Issue by ReisenII** [Nandaka/NijieDownloader#58]
![immagine](https://user-images.githubusercontent.com/26198798/41507730-1e70f978-7239-11e8-94f4-b2fa36a8a3cf.png)

Every time I try to batch, for some (not all) image sets, the app downloads one image and then stop giving me this error. Translates as "String not recognized as valid DateTime"

I can download still, but I have to restart the whole batch for each image.

**Comment by Nandaka** [comment]
Can you upload the log file (e.g. Batch Download on 2018-06-17.txt) in the application folder?

I tried to download member id: 153227 and it working as expected.

**Comment by ReisenII** [comment]
The file is just a bunch of file locations.

[Batch Download on 2018-06-17.txt](https://github.com/Nandaka/NijieDownloader/files/2109049/Batch.Download.on.2018-06-17.txt)

Halfway I tried to move the destination folder, but that didn't change anything.

**Comment by Nandaka** [comment]
Ya, I give you the wrong filename, should be log-yyyy-mm-dd.txt (e.g log-2018-06-17.txt) for the actual log file.

**Comment by Nandaka** [comment]
Can you try to replace the NijieDownloader.Library.dll and upload the log file again?
I add additional logging to check the actual datetime string the server sent.

[NijieDownloader.Library.zip](https://github.com/Nandaka/NijieDownloader/files/2117430/NijieDownloader.Library.zip)

**Comment by ReisenII** [comment]
Sure thing

I redownloaded the 7z, unpacked, replaced the dll you asked for and tried to run the batch a couple of times like the last time.

[log-2018-06-20.txt](https://github.com/Nandaka/NijieDownloader/files/2118484/log-2018-06-20.txt)

**Comment by Nandaka** [comment]
weird, it failed to parse these dates:
![image](https://user-images.githubusercontent.com/836867/41653797-c15239e0-74b9-11e8-9fcf-a9b0c55cd3e0.png)

not sure why yet, worst case I need to suppress this.

**Comment by Nandaka** [comment]
Can you try one?
[NijieDownloader.Library.zip](https://github.com/Nandaka/NijieDownloader/files/2122122/NijieDownloader.Library.zip)

**Comment by ReisenII** [comment]
(As before, redownloaded the 7z, unpacked, replaced the dll file)
The batch no longer stops and I could download the whole user id 153227 in one go, although the single downloads felt slower than usual.

The log
[log-2018-06-21.txt](https://github.com/Nandaka/NijieDownloader/files/2123229/log-2018-06-21.txt)

**Comment by Nandaka** [comment]
yeah, not sure why it still failed to parse the datetime, it looks like the correct format to me.
![image](https://user-images.githubusercontent.com/836867/41728570-b2cf59ac-75a9-11e8-94cb-c34205f7f0ca.png)

Can you set the windows language to english and reupload the log file?

**Comment by ReisenII** [comment]
Looks like the errors are gone with english as system language
[log-2018-06-21.txt](https://github.com/Nandaka/NijieDownloader/files/2124319/log-2018-06-21.txt)
For reference, I have my system in italian.

**Comment by Nandaka** [comment]
ok, I'll try to compile the dll with CultureInfo.InvariantCulture and upload it here.

**Comment by Nandaka** [comment]
Try this one?
[NijieDownloader.Library.zip](https://github.com/Nandaka/NijieDownloader/files/2126313/NijieDownloader.Library.zip)

**Comment by ReisenII** [comment]
Looks like it works as intended now, no errors and a quick uninterrupted download
[log-2018-06-22.txt](https://github.com/Nandaka/NijieDownloader/files/2127055/log-2018-06-22.txt)
